### ISSUE TITLE
Subscribe: after Errors are triggered, focus order isn't consistent

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#136]
URL: https://recalls-rappels.canada.ca/en/subscribe
User Agent: Chrome, Firefox, Edge, JAWS and NVDA
Language: both

Issue Description:  
after Errors are triggered:
- NVDA/Chrome: Screen reader announces "Error Message alert" and focus is randomly placed on a checkbox. Sometimes the focus is moved back to the top of the page (skip links) 
- JAWS/Chrome/EDGE/Firefox: Reads all error messages in list - keyboard focus is at to the top/skip links
- NVDA/Firefox: Focus remains on the Submit button after errors are triggered.
- NVDA/EDGE:  Only reads the Error Message alert. Focus is inconsistent. Sometimes stays on the Submit button, other times on a checkbox and sometimes at the top of the page. 

Steps to reproduce:
1. With a screen reader turned on, from the Subscribe page, trigger the Submit button without completing the required fields. 
2. Using your keyboard, press tab. 
 
Expected results: The experience should be consistent. This could be solved if you applied the WET Error summary where the focus would be placed on the Error summary.

**Comment by davidgotlieb** [comment]
@mmiron-HC 
Similar to #105:

|We are using the default WxT error handling. From OP:

WxT is not 100% WET compliant. Currently, error message handling in WxT is not done the same as it WET. It hasn't been implemented yet. It will be a major undertaking to get the WET error message handling into WxT, when it does get there, it will be pulled into RSAMS as part of the regular WXT update.

**Comment by mmiron-HC** [comment]
@zimw7 @40999221 @YuxuanXie01  Can you please retest with NVDA and confirm if the issues persists?

**Comment by 40999221** [comment]
Retested the issue, it still persist. 
Checked with NVDA for all three browsers for few times, here are the findings:

NVDA/EDGE: Read error message alert, then focus moved to checkbox.
NVDA/Chrome: Read error message alert, then focus moved to checkbox.
NVDA/Firefox: Not reading error message alert, screen reader in browse mode and reading the page

**Comment by zimw7** [comment]
Issue still exists:
NVDA/Chrome: Screen reader announces "Error Message alert". After the first activation of the submit button, focus is randomly placed on a checkbox. Focus remains on the submit button after errors are triggered.
NVDA/Firefox: Not reading error message alert. After the first activation of the submit button, focus is randomly placed on a checkbox. Focus remains on the submit button after errors are triggered.
NVDA/Edge: Only reads the error message alert. After the first activation of the submit button, focus is randomly placed on a checkbox. Focus remains on the submit button after errors are triggered.

**Comment by mmiron-HC** [comment]
@YuxuanXie01  can you move this over to Jira? thanks.

**Comment by YuxuanXie01** [comment]
Retested, issue persists. 
NVDA/Edge: NVDA announces error messages and sometimes moves the focus to the top of the page, while other times the focus shifts to a random checkbox after activation.
NVDA/Chrome: NVDA announces error messages with the focus sometimes on a random checkbox or at the bottom of the page.
NVDA/Firefox: NVDA fails to announce error messages with the focus ending up on a random checkbox.
